.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,3 +16,18 @@ Fixes #
 
 
 ### Proposed changes in this pull request
+
+
+### Documentation checklist
+
+* [ ] I have added a line to CHANGELOG.txt
+* [ ] I have updated the release notes document
+    * [ ] general notes
+    * [ ] bug-fixes
+    * [ ] improvements/new features
+    * [ ] backwards-imcompatible changes
+    * [ ] required upgrade steps
+    * [ ] names of community contributors
```

Comment:
what would be an example of this?

Reply:
Actually I think it should just be all contributors with commits in that particular release, in the form
 ```
* Daniele Procida <http://github.com/evildmp/>
```